site/js-untar.d.ts

```diff
@@ -0,0 +1,16 @@
+declare module "js-untar" {
```

Comment:
Does js-untar not come with a type definition file, so you had to write one?

Reply:
Exactly